proposed/http-message.md

```diff
@@ -383,6 +383,18 @@ interface MessageInterface
     public function hasHeader($name);
 
     /**
+     * Retrieves a header by the given case-insensitive name as an array of
+     * strings.
+     *
+     * If the header did not appear in the message, this method should return an
```

Comment:
If the header does not appear in the message, this method MUST return an empty array.